### ISSUE TITLE
Terminal output path text does not meet color contrast ratio

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52026]
Environment Details:
VSCode Version : 1.24.1
OS Version : Win10

Additional Details:
MAS Violated: MAS1.4.3

Repro Steps:
1)Launch VS Code.
2) Click on View menu item and navigate to Problems
3)Open Color contrast analyser and check the ratio of paths(C:\Users\v-chranj\)

Actual:
Color contrast ratio is 2.2:1 of paths written beside json files,which is less than required.
Same issue has been observed in below themes as well:
Light
Light+
Quiet Light
Dark+
Kimbie Dark
Monokai
Solarized Dark
Tomorrow Night Blue

Expected:
Color Contrast ratio should be equal to or greater than 4.5:1

Recommendations:
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

User Impact:
The user will not be able to read/see the texts of path.

MAS Reference:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}

Attachment for reference: 
![a11y vscode problems path color contrast](https://user-images.githubusercontent.com/39204909/41466478-11f2efd2-70c0-11e8-8c93-bc7b2ec5270b.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by misolori** [comment]
This was reverted with https://github.com/Microsoft/vscode/commit/b887217768cc2bca5892dbb054fd5a5aba1dbcf3. We should look into changing the opacity when it fails (i.e. on selection).

**Comment by misolori** [comment]
Opened #56484 instead
